EMS/helpers/src/File/TempFile.php

```diff
@@ -14,47 +14,60 @@ private function __construct(public readonly string $path)
     {
     }
 
-    private static function create(): self
+    private static function create(?string $cacheFolder = null): self
```

Comment:
```suggestion
    public static function create(?string $cacheFolder = null): self
```

Reply:
Would also make this one public, for better auto completion when using the TempFile